crates/ruff_linter/src/rules/ruff/rules/incorrectly_parenthesized_tuple_in_subscript.rs

```diff
@@ -64,6 +64,10 @@ pub(crate) fn subscript_with_parenthesized_tuple(checker: &mut Checker, subscrip
     if tuple_subscript.parenthesized == prefer_parentheses || tuple_subscript.elts.is_empty() {
         return;
     }
+    let semantic = checker.semantic();
+    if semantic.in_annotation() || semantic.in_type_definition() {
```

Comment:
Should we also do `semantic.in_simple_string_type_definition()` here?

Reply:
It's not needed -- if the `SIMPLE_STRING_TYPE_DEFINITION` flag has been set on the semantic model, the `TYPE_DEFINITION` flag will also have been set:

https://github.com/astral-sh/ruff/blob/c4e651921ba1911a04102e2052c9fe56f185309d/crates/ruff_linter/src/checkers/ast/mod.rs#L2196-L2204